Makefile

```diff
@@ -16,14 +17,54 @@ start: #docker-compose up -d
 
 bootstrap: init build start
 
-monitor: #by default it logs everything, to log some containers only, use the script by hand
-	./helper.sh --monitor
-
 refresh:  build
 
 live_refresh:  # docker-compose up -d --build (CD TARGET)
 	./helper.sh --refresh
 
+list: #docker-compose ps
+	./helper.sh -v --list
+
+debug:
+	./helper.sh -v --debug
+
+config: debug
+
+monitor: #by default it logs everything, to log some containers only, add the name
+	./helper.sh --monitor
+
+stop: # docker-compose stop
+	./helper.sh --stop
+
+down: # docker-compose down
+	./helper.sh --down
+
+restart:
+	./helper.sh --down
+
+hard_restart: nuke_dev restart
+
+nuke_dev:   #docker-compose down -v #TODO if needed: make this nuke take into account the .env like in deploy
+	./helper.sh --nuke
+
+# Cleanup
+clean_docker_dangling_images:
+	docker rmi $(docker images -qf "dangling=true")
+
+clean_docker_images:
+	docker rmi $(docker images -q)
+
+clean: clean_docker_images clean_docker_dangling_images
+
+prune: clean
+	docker system prune -y
```

Comment:
`-a` maybe? this is what I do all the time

Reply:
that's not a good command imho, "too much" cleaning

Reply:
you do it only because we are forced to, being desperate for disk space